.github/workflows/main.yml

```diff
@@ -68,9 +68,10 @@ jobs:
         path: pronto_mia/coverage/
         retention-days: 2
         
-  build:
+  build-android:
     runs-on: ubuntu-latest
     needs: test
+    if: github.ref == 'refs/heads/master'
```

Comment:
This job was initially added to test if any build works. Shouldn't we check that on every push?